std/assembly/math.ts

```diff
@@ -423,7 +423,9 @@ export namespace NativeMath {
       hi = x;
     } else return 1.0 + x;
     var xx = x * x;
-    var c = x - xx * (P1 + xx * (P2 + xx * (P3 + xx * (P4 + xx * P5))));
+    // var c = x - xx * (P1 + xx * (P2 + xx * (P3 + xx * (P4 + xx * P5))));
+    var xx2 = xx * xx;
```

Comment:
Should either make it `xx` and `xxxx` or `x_2` and `x_4` to avoid confusion.

Reply:
What about `xp2` and `xp4`? I want keep names shortest as much as possible

Reply:
Sure, that's fine as well

Reply:
Though, the other two resemble a mathematical formula a bit more hmm ^^

Reply:
I see better way. Using `s` (square) and `q` (quadruple)

Reply:
And this notation already used in this [line](https://github.com/AssemblyScript/assemblyscript/pull/311/files#diff-4f1e427ed70aece4d8c8162e9b85ad54L466)